dists/scummvm.appdata.xml.cpp

```diff
@@ -21,7 +21,7 @@
 
```

Comment:
Shouldn't we rename this file as well (to _org.scummvm.scummvm.appdata.xml.cpp_)?

Reply:
Done.
In the first time I didn't want to because I am not sure about how weblate will act but as I rename the appdata to metainfo this is too much to keep.